tests/Integration/Behaviour/Features/Scenario/Product/Shop/shop_management.feature

```diff
@@ -410,18 +410,10 @@ Feature: Copy product from shop to shop.
     And I add new image "image1" named "app_icon.png" to product "graphicProduct" for shop "shop1"
     And I add new image "image2" named "some_image.jpg" to product "graphicProduct" for shop "shop1"
     And I copy product graphicProduct from shop shop1 to shop shop2
-    Then product "graphicProduct" should have following images for shop "shop1":
+    Then product "graphicProduct" should have following images for shop "shop1, shop2, shop3, shop4":
```

Comment:
Hmm about this one, I don't think we should be able to get image for shop3 and shop4 since the product is not associated to those shops, we probably should add a check in the query that asserts the request productId is associated to the requested shopId and throw a ProductShopAssociationNotFound exception when it's not